region/brinstar/green/Etecoon Energy Tank Room.json

```diff
@@ -2089,6 +2139,62 @@
       "name": "Base",
       "requires": [],
       "flashSuitChecked": true
+    },
+    {
+      "link": [7, 7],
+      "name": "Use Flash Suit - R-Mode Spark Interrupt (Gain Blue Suit)",
+      "requires": [
+        {"obstaclesCleared": ["R-Mode"]},
+        {"or": [
+          {"obstaclesCleared": ["A"]},
+          {"enemyKill": {
+            "enemies": [["Beetom"]],
+            "excludedWeapons": ["Bombs"]
+          }},
+          "canTrickyDodgeEnemies"
+        ]},
+        {"refill": ["Energy"]},
+        {"useFlashSuit": {}},
+        {"autoReserveTrigger": {"maxReserveEnergy": 95}},
+        "canRModeSparkInterrupt"
+      ],
+      "resetsObstacles": ["R-Mode"],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
+      "note": [
+        "The flash suit must be gained in-room due to R-Mode. Refer to the 'Gain Flash Suit (Spikesuit)' strat for how to do this.",
```

Comment:
It's ok for now but usually I wouldn't recommend referring to another strat by name, because it's likely to become outdated